Specs/DateTime/English/DatePeriodParser.json

```diff
@@ -735,26 +735,26 @@
     ]
   },
   {
-    "Input": "this week",
+    "Input": "The union suspended strike action this week.",
     "Context": {
-      "ReferenceDateTime": "2016-11-07T00:00:00"
+      "ReferenceDateTime": "2026-01-01T00:00:00"
```

Comment:
Don't modify tests like this. If you need to confirm a new scenario, let's add a new sentence for a new reference date.
Otherwise we may loose the original regression check.